execution/evm/vm.go

```diff
@@ -493,6 +493,75 @@ func (vm *VM) call(caller acm.Account, callee acm.MutableAccount, code, input []
 			stack.Push64(int64(res))
 			vm.Debugf(" => 0x%X\n", res)
 
+		case SHL: //0x1B
+			shift, popErr := stack.PopU64()
+
+			if popErr != nil {
+				return nil, firstErr(err, popErr)
+			}
+
+			sh := uint(shift)
```

Comment:
If uint is 32 bit, then this statement truncates. Best to compare shift to 256, and cast to uint on the call to value.Lsh()